### ISSUE TITLE
Setup documentation needed

### ISSUE THREAD

**Issue by Jichao-Yang** [LighghtEeloo/lala-lang#22]
It'd be nice if the author could add to the readme file the instructions to set up the environment for the language. For example, what dependencies are required? What command should I run if I want to write a helloworld file? etc.

**Comment by LighghtEeloo** [comment]
You are right. As the language is currently under heavy development, the document and apis may experience huge changes from time to time. The documents and instructions will be offered both in readme and in github pages as soon as the basic features are stable.
